tests/foreman/api/test_product.py

```diff
@@ -171,25 +167,17 @@ def test_repositoryset_disable(self):
         """
         org = entities.Organization().create()
         org.upload_manifest(path=manifests.clone())
-        prd_id = entities.Product().fetch_rhproduct_id(
+        product = entities.Product(
             name=PRDS['rhel'],
-            org_id=org.id,
-        )
-        product = entities.Product(id=prd_id)
-        reposet_id = product.fetch_reposet_id(name=REPOSET['rhva6'])
-        product.enable_rhrepo(
-            base_arch='x86_64',
-            release_ver='6Server',
-            reposet_id=reposet_id,
-        )
-        product.disable_rhrepo(
-            base_arch='x86_64',
-            release_ver='6Server',
-            reposet_id=reposet_id,
-        )
-        repositories = product.repository_sets_available_repositories(
-            reposet_id=reposet_id,
-        )
+            organization=org,
+        ).search()[0]
+        reposet = entities.RepositorySet(
+            name=REPOSET['rhva6'],
+            product=product,
+        ).search()[0]
+        reposet.enable({'basearch': 'x86_64', 'releasever': '6Server'})
+        reposet.disable({'basearch': 'x86_64', 'releasever': '6Server'})
```

Comment:
As per meeting discussion today, Can we create a robottelo helper method for this?